protocol/0028-GOVE-governance.md

```diff
@@ -546,8 +546,8 @@ APIs should also exist for clients to:
 - A market change proposal that's to modify any parameters on a market in `pending` state (i.e. voting has successfully completed and the market is in the opening auction) will be accepted and if it's the enactment time happens to be before the opening auction ends then the proposed modification is enacted. (<a name="0028-GOVE-070" href="#0028-GOVE-070">0028-GOVE-070</a>)
 - In particular a market change proposal that's to modify the parent market on a market in `pending` state (i.e. voting has successfully completed and the market is in the opening auction) will be accepted and if it's the enactment time happens to be before the opening auction ends then the parent is used (assuming the proposed parent doesn't already have a successor). (<a name="0028-GOVE-071" href="#0028-GOVE-071">0028-GOVE-071</a>)
 - A market change that's to modify any parameters on a market in `pending` state (i.e. voting has successfully completed on the market creation and the market is in the opening auction) will run voting rules the same as market creation proposals i.e. LPs don't get a vote. (<a name="0028-GOVE-072" href="#0028-GOVE-072">0028-GOVE-072</a>)
-- A market change proposal that aims to modify the market name in any state will modify the market name at the time of vote enactment. (<a name="0028-GOVE-159" href="#0028-GOVE-159">0028-GOVE-159</a>)
-- A market change proposal that aims to modify the market code in any state will modify the market code at the time of vote enactment. (<a name="0028-GOVE-166" href="#0028-GOVE-166">0028-GOVE-166</a>)
+- A market change proposal that aims to modify the market name in any state (apart from closed/terminated) will modify the market name at the time of vote enactment. (<a name="0028-GOVE-159" href="#0028-GOVE-159">0028-GOVE-159</a>)
```

Comment:
Can you check with core whether the ` (apart from closed/terminated) ` makes life easier or harder? In a way I don't care - if someone renames a closed market - that's their job.

Reply:
Its already implemented by Zohar like this :)